hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/SyntheticTxnFactory.java

```diff
@@ -501,6 +504,42 @@ public TransactionBody.Builder createUnFreeze(
         return TransactionBody.newBuilder().setTokenUnfreeze(builder);
     }
 
+    public TransactionBody.Builder createTokenUpdate(TokenUpdateWrapper updateWrapper) {
+        final var builder = TokenUpdateTransactionBody.newBuilder();
+        builder.setToken(updateWrapper.tokenID())
+                .setName(updateWrapper.name())
```

Comment:
What if I don't want to change the name, symbol, or memo.  Do I have to query it and set it in the wrapper for the call?

I think it should get the same null handling treasury, expirey, et.al. get.  If there some weird protobuf "null is indistingusiable from usnet" things going on that should be called out in a comment.